http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/NodepoolComponent.scala

```diff
@@ -165,6 +172,15 @@ object nodepoolQuery extends TableQuery(new NodepoolTable(_)) {
       nodepools <- findByNodepoolIdQuery(id).result
     } yield nodepools.map(rec => unmarshalNodepool(rec, List())).headOption
 
+  def getMinimalByUser(creator: WorkbenchEmail,
```

Comment:
does the returned `Nodepool` one that has some fields filled with dummy values? if yes, I think we should move away from that and create a custom data type for the returned type. If not, we can probably just call this `getNodepoolByUser`

Reply:
This is the naming convention the nodepool queries use to distinguish whether or not associated apps are returned (an empty list will be returned here for apps)